middlewares/error-middleware.js

```diff
@@ -30,8 +30,9 @@ const errorMiddleware = (err, req, res, next) => { // eslint-disable-line no-unu
       const regexResult = pathQueryRegex.exec(path);
       const formattedPath = regexResult ? regexResult[1] : path;
 
-      if (path === 'studentID' && errorCode === 'pattern.openapi.validation') {
-        details.push('studentID must be 9 digits');
+      if (_.includes(['studentID', 'creatorID'], path)
+        && errorCode === 'pattern.openapi.validation') {
```

Comment:
Just my two cents, instead of forcing to break the condition as two lines, I guess it will be better to have a variable to store all parameters which should be validated as 9 digits ID, for example:

```js
const nineDigitsIDs = ['studentID', 'creatorID'];
```

I guess it can help the readability a bit and also allow us to add more criteria if needed in the future.